ebr_trackerbot/command/show.py

```diff
@@ -0,0 +1,87 @@
+"""
+Slack Bot Track Command
+"""
+
+import re
+import logging
+import requests
+from bot import register_command
+import pendulum
+
+
+def show_command(text, result, payload, config, commands):
+    """
+    Slack Bot Show Command
+    """
+    logging.debug("Show command")
+
+    target_test = result.group(1)
+    duration = result.group(2)
+    start = pendulum.now("UTC")
+    parts = re.split("([smhdy])", duration)
+    number = None
+    for part in parts:
+        if part in ["s", "m", "h", "d"] and number is not None:
+            if part == "s":
+                start = start.subtract(seconds=number)
+            if part == "m":
+                start = start.subtract(minutes=number)
+            if part == "h":
+                start = start.subtract(hours=number)
+            if part == "d":
+                start = start.subtract(days=number)
+        else:
+            try:
+                number = int(part)
+            except ValueError:
+                number = None
+
+    response = requests.get(
+        config["api_url"],
+        params={
+            "test_status": "failed",
+            "start": start.to_iso8601_string(),
+            "end": pendulum.now("UTC").to_iso8601_string(),
+        },
+        headers={"accept": "application/json"},
+    )
+
+    channel_id = payload["data"]["channel"]
```

Comment:
Is it guaranteed that these items are always in the payload?

Reply:
Yes, this comes from slack.